README.md

```diff
@@ -84,18 +84,26 @@ The following prerequisites are required:
 
 
 
-Open a terminal, go into the `coatrack` directory, build all components and run the Web Application component (admin):
+Open a terminal, go into the `coatrack` directory and build all components using one the the commands below:
+
+```sh
+bash build.sh # For Linux shells
+mvn clean package # For Linux or Windows shells
```

Comment:
I assume the `build.sh` does not work for windows? In this case it does not make sense to keep it here, as the second command should work for everyone...